packages/tkg-clusterclass-aws/bundle/config/upstream/overlay-podsecuritystandard.yaml

```diff
@@ -0,0 +1,115 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.subset({"kind":"ClusterClass"})
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+spec:
+  #@overlay/match missing_ok=True
+  variables:
+  #@overlay/remove
+  #@overlay/match by=overlay.map_key("name"),missing_ok=True
+  - name: podSecurityStandard
+  #@overlay/append
+  - name: podSecurityStandard
+    required: false
+    schema:
+      openAPIV3Schema:
+        type: object
+        default: {}
+        properties:
+          disabled:
```

Comment:
Under inclusive naming guidelines, I think we prefer to use deactivated/activated instead of disabled/enabled, for any code we control.

Reply:
Great point 👍  I refactored it to deactivated.